user/reference/overview.md

```diff
@@ -17,15 +17,46 @@ Each build runs in one of the following virtual environments.
 
-* [Ubuntu Focal 20.04](/user/reference/focal/)
-* [Ubuntu Bionic 18.04](/user/reference/bionic/)
-* [Ubuntu Xenial 16.04](/user/reference/xenial/) **default**
-* [Ubuntu Trusty 14.04](/user/reference/trusty/)
-* [Ubuntu Precise 12.04](/user/reference/precise/)
+#### Full VM
 
-LXD compliant OS images for arm64 are run in [Packet](https://www.packet.com/). Have a look at [Building on Multiple CPU Architectures](/user/multi-cpu-architectures) for more information.
+This is sudo enabled, full virtual machine per build, that runs Linux
+
+* Spins up slower (increased build time compared to LXD container) yet without any [limitations](/user/reference/overview/#linux-security-and-lxd-container).
+* It has a fixed amount of vCPUs and RAM assigned.
+
+#### LXD container
+
+This is sudo enabled LXD container build environment, as close to a virtual machine as you can get in containers world. A Linux environment is run within an unprivileged LXD container. 
+
+* Fast spin-up (decreased build time when compared to full VM) yet some [limitations](/user/reference/overview/#linux-security-and-lxd-container) do apply
+* It starts with min 2 vCPUs and if there is more computing time available, the host can dynamically assign it to speed up your build
+
+#### Which one do I use?
+
+For the majority of cases, whenever available, we recommend to use LXD-based containers. 
+
+Use full VM only if LXD is not available or you need 
+* privileged fs access
+* specific system call interception
+* hugepages support (subject to changes on short notice)
+
+Below table sums up available Ubuntu environments and virtualization type per CPU architecture:
+
+| Ubuntu version       | Linux Virtualization Type    |
+| :------------------- | :---------------------       |
+| [Ubuntu Focal 20.04](/user/reference/focal/) | `arch: amd64`: full VM only, default option<br />`arch: arm64`: LXD only<br />`arch: arm64-graviton2`: LXD and full VM<br/>`arch: ppc64le`: LXD only<br/>`arch: s390x`: LXD only |
+| [Ubuntu Bionic 18.04](/user/reference/bionic/) | `arch: amd64`: full VM only, default option<br />`arch: arm64`: LXD only<br />`arch: arm64-graviton2`: LXD only<br/>`arch: ppc64le`: LXD only<br/>`arch: s390x`: LXD only  |
+| [Ubuntu Xenial 16.04](/user/reference/xenial/) **default** | `arch: amd64`: full VM only, default option<br />`arch: arm64`: LXD only<br />`arch: arm64-graviton2`: LXD only<br/>`arch: ppc64le`: LXD only<br/>`arch: s390x`: LXD only  |
+| [Ubuntu Trusty 14.04](/user/reference/trusty/) | `arch: amd64`: full VM only, default option  |
+| [Ubuntu Precise 12.04](/user/reference/precise/) | `arch: amd64`: full VM only, default option  |
+
+
+LXD compliant OS images for arm64 are run on [AWS](https://aws.amazon.com/) and in [Packet](https://www.packet.com/). LXD compliant OS images for IBM Power and Z are run in [IBM Cloud](https://www.ibm.com/cloud). Have a look at [Building on Multiple CPU Architectures](/user/multi-cpu-architectures) for more information.
```

Comment:
**For more information see** Building on ...

Reply:
updating on master, thanks!